src/webui/www/private/css/Layout.css

```diff
@@ -101,11 +86,12 @@ body {
 
 /* Navbar */
 
+#Filters_pad {
+    padding: 0 1px!important;
```

Comment:
You should set them at: https://github.com/qbittorrent/qBittorrent/blob/0f40fad74d6e366b16195bbf89659a0eb1cd131e/src/webui/www/private/scripts/client.js#L300-L304